src/EventId.php

```diff
@@ -35,7 +35,7 @@ public function __construct(string $value)
      */
     public static function generate(): self
     {
-        return new self(str_replace('-', '', uuid_create(UUID_TYPE_RANDOM)));
+        return new self(str_replace('-', '', uuid_create(\UUID_TYPE_RANDOM)));
```

Comment:
unrelated change to the pull request. should be submitted separately

Reply:
and since already nit-picking about commits quality, having commit titled "CR fixes" does not give any value to commit history, since you already rebased and force pushed, squash it with original commit.

Reply:
>should be submitted separately

This change actually is unwanted, it seems that when `ext-uuid` extension is installed (like on my environment) `PHP-CS-Fixer` consider the constant as available natively while when not present (but the polyfill is) it doesn't.